flutter_vlc_player/example/lib/single_tab.dart

```diff
@@ -92,6 +97,14 @@ class _SingleTabState extends State<SingleTab> {
             advanced: VlcAdvancedOptions([
               VlcAdvancedOptions.networkCaching(2000),
             ]),
+            subtitle: VlcSubtitleOptions([
+              VlcSubtitleOptions.boldStyle(true),
+              VlcSubtitleOptions.fontSize(30),
+              // works only on external subs
+              VlcSubtitleOptions.color(VlcSubtitleColor.Navy),
+              // works only on external subs
+              VlcSubtitleOptions.outlineColor(VlcSubtitleColor.Yellow),
```

Comment:
Outline color changes with built-in subs for me. What platform are you testing on?

Reply:
yea, you'r right.